### PR TITLE
Fix bug in lr scheduler configure

### PR DESCRIPTION
**Pull request by EnricoTrizio**
## Description
Fix bug in lr_scheduler in configure_optimizer in the base CV class to solve #115

## Todos
  - [ ] Add tests

## Status
- [ ] Ready to go

### CODE REVIEW

**Review by andrrizzi — APPROVED** [review]
Looks good to me!
